.travis.yml

```diff
@@ -83,6 +83,7 @@ cache:
     - $HOME/.cache/.rotkehlchen-test-dir
     - $HOME/.ethash
     - $HOME/.bin
+    - $HOME/.cache/Cypress
```

Comment:
@LefterisJP Added this to save some time with Cypress

Reply:
nice thanks! Does Github actions also have this?

Reply:
Yes I first added it there